build.cake

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 // TOOLS
 ///////////////////////////////////////////////////////////////////////////////
-#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.1.20160523.144749
+#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.2.20160623.172854-eap5"
```

Comment:
maybe it's because it's an prerelase `2016.2.20160623.172854-eap5` worked for me and is the latest.

You could add `&prerelease` though and it might sort it.